zstacklib/zstacklib/utils/ovn.py

```
@@ -299,3 +306,184 @@ def addUplink(self, portPciMap, bondMode, lacpmode, ip, netmask, brName="br-phy"
+            if os.path.exists(VSWITCHD_PID_PATH):
+                with open(VSWITCHD_PID_PATH, 'r') as f:
+                    pids[1] = int(f.read().strip())
+
+            if os.path.exists(OVN_CONTROLLER_PID_PATH):
+                with open(VSWITCHD_PID_PATH, 'r') as f:
+                    pids[2] = int(f.read().strip())
+
+        except OSError as err:
+            logger.error("OSError: {}".format(err))
+        finally:
+            return pids[0] != -1 and pids[1] != -1 and pids[2] != -1
```

Ruff (0.11.9)

`return` inside `finally` blocks cause exceptions to be silenced

(B012)